src/DependenciesPackager/Program.cs

```diff
@@ -0,0 +1,235 @@
+                        throw new InvalidOperationException("Error restoring nuget packages into destination folder");
+                    }
+
+                    RemoveUnnecessaryFiles();
+                    CreateZipPackage();
+                }
+                catch (Exception e)
+                {
+                    Logger.LogError(e.Message);
+                    return Error;
+                }
+
+                return Ok;
+            }
+            catch (Exception e)
+            {
+                Logger.LogError(e.Message);
+                _app.ShowHelp();
+                return Error;
+            }
+        }
+
+        private void CreateZipPackage()
+        {
+            var packagesFolder = Path.Combine(_destination.Value(), _version.Value());
+            var zipFileName = Path.Combine(_destination.Value(), $"AspNetCore.{_version.Value()}.zip");
+            Logger.LogInformation($"Creating zip package on {zipFileName}");
+            ZipFile.CreateFromDirectory(packagesFolder, zipFileName);
+        }
+
+        private void RemoveUnnecessaryFiles()
+        {
+            Logger.LogInformation($"Removing unnecessary files from folder");
+            var packagesFolder = Path.Combine(_destination.Value(), _version.Value());
+            var dlls = Directory.GetFiles(packagesFolder, "*.dll", SearchOption.AllDirectories);
+            var shas = Directory.GetFiles(packagesFolder, "*.sha512", SearchOption.AllDirectories);
+            var allEntries = Directory.GetFiles(packagesFolder, "*", SearchOption.AllDirectories);
+            var entriesToRemove = allEntries.Except(dlls).Except(shas);
+
+            foreach (var entry in entriesToRemove)
+            {
+                File.Delete(entry);
+            }
+        }
+
+        private int RunDotnetRestore(string projectJson)
+        {
+            Logger.LogInformation($"Running dotnet restore on {Path.Combine(_destination.Value(), _version.Value())}");
+            var sources = string.Join(" ", _sourceFolders.Values.Select(v => $"--source {v}"));
+            var fallbackFeeds = string.Join(" ", _fallbackFeeds.Values.Select(v => $"--fallbacksource {v} "));
+            var packages = $"--packages {Path.Combine(_destination.Value(), _version.Value())}";
+
+            var dotnet = _dotnetPath.HasValue() ? _dotnetPath.Value() : "dotnet";
+
+            var arguments = string.Join(" ",
+                "restore",
+                packages,
+                projectJson,
+                sources,
+                fallbackFeeds,
+                "--verbosity Verbose");
+
+            var processInfo = new ProcessStartInfo(dotnet, arguments);
+            processInfo.CreateNoWindow = true;
+            processInfo.UseShellExecute = false;
+            processInfo.RedirectStandardError = true;
+            processInfo.RedirectStandardOutput = true;
+            var process = Process.Start(processInfo);
+
+            string line = process.StandardOutput.ReadLine();
```

Comment:
I've changed the implementation on the PR for 49, it's based on events, so I will not change it here since I'll do the integration with the build after both things are in.